src/mutexes/Redis.ts

```diff
@@ -15,12 +15,12 @@ declare module 'ioredis' {
 
 export default class RedisMutex extends RatelimitMutex {
 	public static readonly keys = {
-		global: 'global',
-		remaining: (route: string) => `${route}:remaining`,
-		limit: (route: string) => `${route}:limit`,
+		global: (prefix?: string) => `${ prefix ? `${ prefix }:` : '' }global`,
+		remaining: (route: string, prefix?: string) => `${ prefix ? `${ prefix }:` : '' }${route}:remaining`,
+		limit: (route: string, prefix?: string) => `${ prefix ? `${ prefix }:` : '' }${route}:limit`,
```

Comment:
I'm thinking that it might be better to make these into an instance property now that they are related to instance data. We could then avoid having to pass `this.prefix` to all the key creators.

Reply:
That was my thought process, figured there was a reason for making them static though, will fix.